fluent_comments/templatetags/fluent_comments_tags.py

```diff
@@ -145,7 +145,8 @@ def get_template_name(self, *tag_args, **tag_kwargs):
         return get_comment_template_name(comment=tag_args[0])
 
     def get_context_data(self, parent_context, *tag_args, **tag_kwargs):
-        return get_comment_context_data(comment=tag_args[0])
+        _, request, _ = parent_context
```

Comment:
This can't possibly work on Python 2.
How about `request = parent_context.get('request')`